api/src/main/java/org/pmiops/workbench/workspaces/resources/UserRecentResourceServiceImpl.java

```diff
@@ -64,13 +64,13 @@ public DbUserRecentResource updateNotebookEntry(
       handleUserLimit(userId);
       recentResource = new DbUserRecentResource(workspaceId, userId, notebookNameWithPath, now);
     }
-    recentResource.setLastAccessDate(now);
-    updateUserRecentlyModifiedResourceEntry(
+    userRecentResourceDao.save(recentResource.setLastAccessDate(now));
```

Comment:
still saves the old version and the new version.

the difference is that we now return the new version.